automatic/poi/poi.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
+    <id>poi</id>
+    <version>9.0.0</version>
+    <owners>chocolatey, POI Contributors</owners>
```

Comment:
POI Contributors in this case should be replaced with the user you have on chocolatey, your name or your github username.
This is meant as the owner of the package, not the software.

Reply:
Fixed in 80c1b3e